plugins/dfu/dfu.quirk

```diff
@@ -431,7 +431,7 @@ Flags = manifest-poll,allow-zero-polltimeout,unsigned-payload
 RemoveDelay = 9000
 
 # AVer ATLAS CAM
-[USB\VID_2574&PID_0A70]
+[USB\VID_34AD&PID_0006]
```

Comment:
Should both IDs be added or is this some circumstance that only the new one is needed?

Reply:
If you want to add both that's fine; just do a new PR and we can have the old and new ones.